### PR TITLE
Speculative blog post about the future and funding of science

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

**Comment by gvwilson** [comment]
@ctb could you please have a look?

File: blog/2015/01/future-and-funding-of-science.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  global climate change,
+  resource depletion,
+  drug-resistant diseases,
+  the Sixth Great Extinction,
+  and aging populations in the industrialized world
+  are all going to hit us at more or less the same time.
+  Research that doesn't directly address one or more of these problems
+  will be sidelined in favor of research that does&mdash;basically,
+  trips to Mars and LHC-style "big physics" will be starved
+  so that climatology, materials science, and ecology can be fed.
```
Comment (ctb) [inline blog/2015/01/future-and-funding-of-science.html]: health, too. Not sure why ecology is going to get any feeding? :(

Comment (jiffyclub) [inline blog/2015/01/future-and-funding-of-science.html]: Yeah, definitely need medicine here. And while materials science is the foundation of civilization, there's nothing in the text before here that explicitly relates to it.

File: blog/2015/01/future-and-funding-of-science.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  "grand challenge" initiatives modeled on the Manhattan Project and the Apollo program
+  will receive the lion's share of funding.
+  Many scientists will complain that freedom of inquiry is disappearing,
+  but this doesn't mean that fundamental research will disappear:
+  historically,
+  the Second World War was a tremendously productive period at all levels of scientific inquiry
+  precisely because scientists were working toward specific goals.
+  It does mean,
+  though,
+  that politicians (who represent the interests of the people who fund research)
```
Comment (jiffyclub) [inline blog/2015/01/future-and-funding-of-science.html]: This is an optimistic description of politicians. (:sob:) Isn't lobbying going to continue to run this show?

**Pull request merged**